product_code_builder_sequence/models/product_product.py

```diff
@@ -46,3 +46,18 @@ def copy(self, default=None):
                     'prefix_code': self.prefix_code + _('-copy'),
                 })
         return super(ProductTemplate, self).copy(default)
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    @api.multi
+    def copy(self, default=None):
+        for product in self:
+            if default is None:
+                default = {}
+            if product.prefix_code:
```

Comment:
@gdgellatly Yes, this was a long standing issue since https://bugs.launchpad.net/openobject-addons/+bug/942699 but my fix was finally merged in June this year! https://github.com/odoo/odoo/pull/12397